.gitignore

```diff
@@ -141,121 +73,37 @@ dynawo/3rdParty/libxml/sax/parser/include/xml/sax/parser/ParserVersion.h
-documentation/licenses/jQuery/license-2.log
-documentation/licenses/jQuery/license-2.pdf
-documentation/licenses/jQuery/license-2.synctex.gz
-documentation/licenses/suitesparse/license.aux
-documentation/licenses/suitesparse/license.log
-documentation/licenses/suitesparse/license.pdf
-documentation/licenses/suitesparse/license.synctex.gz
-documentation/licenses/sundials/license.aux
-documentation/licenses/sundials/license.log
-documentation/licenses/sundials/license.pdf
-documentation/licenses/sundials/license.synctex.gz
-documentation/licenses/Adept/license-no-numbering.aux
-documentation/licenses/Adept/license-no-numbering.log
-documentation/licenses/Adept/license-no-numbering.pdf
-documentation/licenses/OpenModelica/license-no-numbering.aux
-documentation/licenses/OpenModelica/license-no-numbering.log
-documentation/licenses/OpenModelica/license-no-numbering.pdf
-documentation/licenses/cpplint/license-no-numbering.aux
-documentation/licenses/cpplint/license-no-numbering.log
-documentation/licenses/cpplint/license-no-numbering.pdf
-documentation/licenses/dynawo-documentation/license-no-numbering.aux
-documentation/licenses/dynawo-documentation/license-no-numbering.log
-documentation/licenses/dynawo-documentation/license-no-numbering.pdf
-documentation/licenses/dynawo/license-no-numbering.aux
-documentation/licenses/dynawo/license-no-numbering.log
-documentation/licenses/dynawo/license-no-numbering.pdf
-documentation/licenses/jQuery/license-1-no-numbering.aux
-documentation/licenses/jQuery/license-1-no-numbering.log
-documentation/licenses/jQuery/license-1-no-numbering.pdf
-documentation/licenses/jQuery/license-2-no-numbering.aux
-documentation/licenses/jQuery/license-2-no-numbering.log
-documentation/licenses/jQuery/license-2-no-numbering.pdf
-documentation/licenses/suitesparse/license-no-numbering.aux
-documentation/licenses/suitesparse/license-no-numbering.log
-documentation/licenses/suitesparse/license-no-numbering.pdf
-documentation/licenses/sundials/license-no-numbering.aux
-documentation/licenses/sundials/license-no-numbering.log
-documentation/licenses/sundials/license-no-numbering.pdf
-documentation/licenses/xerces-c/license-no-numbering.aux
-documentation/licenses/xerces-c/license-no-numbering.log
-documentation/licenses/xerces-c/license-no-numbering.pdf
-documentation/licenses/*/license-no-numbering.aux
-documentation/licenses/*/license-no-numbering.log
-documentation/licenses/*/license-no-numbering.pdf
-documentation/advancedDoc/advancedDoc.aux
-documentation/advancedDoc/advancedDoc.log
-documentation/advancedDoc/advancedDoc.out
-documentation/advancedDoc/advancedDoc.pdf
-documentation/advancedDoc/advancedDoc.synctex.gz
-documentation/advancedDoc/advancedDoc.toc
 util/autocompletion/_dynawo
 .DS_Store
-util/curvesToHtml/resources/1.3.2.tar.gz
-util/curvesToHtml/resources/v0.6.0.tar.gz
 curvesOutput
+!**/reference/
+*.aux
+*.bak-mo
+*.bbl
+*.blg
+*.dvi
+*.gz
+*.log
 *.orig
+*.out
+*.pdf
+*.pyc
+*.toc
+*.zip
```

Comment:
The modifications done here are too generic it might ignore files we need, it's better to leave it as it is.

Reply:
When a .tex file is compiled, several additional files are generated : .aux, .log, .out, .pdf, .synctex.gz and (if there is a table of contents) .toc.

.log and .pdf can be found elsewhere but .aux, .out, .synctex.gz and .toc are specific to a .tex compilation.

Moreover, the .bak-mo files are generated when a .mo is deleted through OpenModelica, they ought not to be saved.